cli/cmd/admin/admin.go

```diff
@@ -7,10 +7,15 @@ import (
 
 // AdminCmd represents the admin command
 func AdminCmd(ch *cmdutil.Helper) *cobra.Command {
+	internalGroupID := ""
+	if ch.IsDev() {
+		internalGroupID = "Internal"
+	}
```

Comment:
You could also consider passing a `hidden bool` to `AddGroup` instead (where if `hidden` is true, it marks the commands hidden and adds them to the root command instead).